packages/zapp/console/src/test/setupTests.ts

```diff
@@ -20,3 +20,9 @@ afterEach(() => {
 afterAll(() => {
   mockServer.close();
 });
+
+global.ResizeObserver = jest.fn().mockImplementation(() => ({
```

Comment:
nit: curious why this is needed?

Reply:
Thank you for explanation. 
Can you addd comment for it, something along the lines : `needed for WorkflowGraph tests`

Reply:
actually, as I'm mocking ReactFlowWrapper now instead of chain rendering, we can get rid of ResizeObserver